includes/ks/icon-row.html

```diff
@@ -0,0 +1,101 @@
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore</p>
+                        <a href="#" class="button button--umkc-yellow button--cta">Button Text</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="content-row bg-umkc-gray">
+    <h2 style="margin-bottom:2em;">Icon Row (4 Up)</h2>
+    <div class="icon-row icon-row--small-stats jc-evenly">
+        <div class="icon-row__group">
+            <div class="icon-row icon-row--small-stats jc-evenly">
+                <div class="icon-row__item icon-row__item--stats">
+                    <div class="icon-row__image">
+                        <img src="src\images\svg\computer-arrow.svg" class="" alt="">
+                    </div>
+                    <div class="icon-row__details">
+                        <a href="#" class="icon-row-title-link" title="">
+                            <h3 class="icon-row__title icon-row__title--light">Point</h3>
+                        </a>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore</p>
+                    </div>
+                </div>
+                <div class="icon-row__item icon-row__item--stats">
+                    <div class="icon-row__image">
+                        <img src="src\images\svg\building-dormApartment.svg" class="" alt="">
+                    </div>
+                    <div class="icon-row__details">
+                        <a href="#" class="icon-row-title-link" title="">
+                            <h3 class="icon-row__title icon-row__title--light">Point</h3>
+                        </a>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore</p>
+                    </div>
+                </div>
+                <div class="icon-row__item icon-row__item--stats">
+                    <div class="icon-row__image">
+                        <img src="src\images\svg\book-academicUnits.svg" class="" alt="">
+                    </div>
+                    <div class="icon-row__details">
+                        <a href="#" class="icon-row-title-link" title="">
+                            <h3 class="icon-row__title icon-row__title--light">Point</h3>
+                        </a>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore</p>
+                    </div>
+                </div>
+                <div class="icon-row__item icon-row__item--stats">
+                    <div class="icon-row__image">
+                        <img src="src\images\svg\books-stacked.svg" class="" alt="">
+                    </div>
+                    <div class="icon-row__details">
+                        <a href="#" class="icon-row-title-link" title="">
```

Comment:
With classing the title (hX), not the link, the default hover behavior should take effect.